### PR TITLE
ZTF upload

### PR DESCRIPTION
**Pull request by profjsb**
- database changes 
- calculate variable star flux (using difference imaging)
- plot updates, with errorbars

### CODE REVIEW

**Comment by profjsb** [comment]
Current SourceList page:
skyportal

Current Source page:
skyportal

**Comment by profjsb** [comment]
This should be close to ready to go. Note:  I left in the work I did to parallelize the reading of AVRO files, but there's some complaints via `concurrent.futures` about pickling of lambda functions (related to the User table). Could not debug this.

File: static/js/components/SourceList.jsx [hunk 1]
```diff
@@ -8,17 +8,31 @@ const SourceList = ({ sources }) => (
...
+              </td>
+              <td>{source.detect_photometry_count}</td>
+              <td>{source.simbad_class}</td>
+          </tr>
         ))
       }
-    </ul>
+    </table>
+    <script>
+    sorttable.makeSortable(document.getElementById('tab'));
```
Comment (stefanv) [inline static/js/components/SourceList.jsx]: What does this do?

**Comment by acrellin** [comment]
Looks good to me so far, but I'm not seeing the associated changes to the Source and Photometry handlers - shouldn't that be included here, or am I missing something?

File: static/js/components/SourceList.jsx (hunk 1 above)
Comment (profjsb) [reply static/js/components/SourceList.jsx]: I was trying to add a sortable table, but failed. Removed this.

**Comment by stefanv** [comment]
@acrellin Scripts that use load_env should allow flags such as `--config` and `--debug`, otherwise there is no way to specify custom configuration.

**Comment by stefanv** [comment]
`--workers` argument broken; does not cast string to int

**Comment by acrellin** [comment]
Closing and moving changes to https://github.com/skyportal/skyportal/pull/104

**Pull request closed without merging**